native/cocos/bindings/manual/Worker.cpp

```diff
@@ -0,0 +1,619 @@
+/**
+ * Created by ihowe@outlook.com on 2023/5/31.
+ * Worker on Native
+ */
+
+#include <chrono>
+#include <thread>
+#include <sstream>
+#include <atomic>
+
+#include "Worker.h"
+// include cocos
+#include <cocos/base/Macros.h>
+#include <cocos/base/Log.h>
+#include <cocos/platform/FileUtils.h>
+#include <cocos/bindings/jswrapper/config.h>
```

Comment:
Use `double quotes`  to include files in engine. And `cocos` prefix is not needed.
```suggestion
#include "bindings/jswrapper/config.h"
```